db/migrate/20240319182716_add_cross_core_project_to_orders.rb

```diff
@@ -0,0 +1,5 @@
+class AddCrossCoreProjectToOrders < ActiveRecord::Migration[7.0]
+  def change
+    add_column :orders, :cross_core_project_id, :integer, null: true
```

Comment:
I suggest using [`add_reference`](https://stackoverflow.com/questions/52915025/add-foreign-key-vs-add-reference-in-rails) here to create an index and foreign key constraints:
```suggestion
    add_reference :orders, :cross_core_project_id, foreign_key: true
```